feat(training-data): expand pre-training and chat training datasets w…

Genstark opened the pull request:
…ith additional examples

Genstark committed:
feat(training-data): expand pre-training and chat training datasets with additional examples

Genstark committed:
Merge branch 'main' into add-data

tekaratzas commented:
Hey! We actually moved training samples into the JSON file in the data folder.

Can you move them there to keep consistent?

Genstark commented:
sure

Genstark committed:
Merge branch 'main' into add-data

Genstark committed:
feat(training-data): add new examples to pre-training and chat training datasets

Genstark committed:
Merge branch 'add-data' of https://github.com/Genstark/RustGPT into add-data

Genstark commented:
done

Genstark commented:
How will this problem be solved?

Genstark committed:
refactor

Genstark commented:
now check if there is any build problem

Genstark committed:
Merge branch 'main' into add-data

tekaratzas commented:
Looks like there is a conflict!

Genstark committed:
Merge branch 'main' into add-data

Genstark commented:
Check now if there is any problem now.

tekaratzas commented:
Looks like a unit test + clippy failure!

can you take a look?

Genstark committed:
fix: correct imports for LLM, Embeddings, Vocab, TransformerBlock, and OutputProjection

Genstark committed:
Merge branch 'add-data' of https://github.com/Genstark/RustGPT into add-data

Genstark commented:
fix: resolve import errors and undeclared types in main.rs and llm.rs

- Corrected use statements to properly import structs and constants from submodules:
  - `Embeddings`, `Vocab`, `TransformerBlock`, `OutputProjection`, `LLM`
  - `EMBEDDING_DIM`, `HIDDEN_DIM`, `MAX_SEQ_LEN`
- Removed incorrect leading colons and fixed capitalization issues
- Ensures all types are correctly recognized by Rust compiler

Genstark commented:
hello tekaratzas can you check current commit working or not

Genstark committed:
refactor: consolidate imports and remove unused module declarations
